### PR TITLE
feat: update mongoose dependency to 7.3.4

### PR DESCRIPTION
**Pull request by keitatata**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,28 +2,28 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.12.11", "@babel/code-frame@^7.0.0":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
+  resolved "https://caminc.jfrog.io/caminc/api/npm/default/@babel/code-frame/-/code-frame-7.12.11.tgz"
```
Comment (hsluoyz) [inline yarn.lock:7]: Don't change our registry to this `caminc.jfrog.io`, we don't use this. This is why causes this CI error:

image


Keep using `registry.npmjs.org` or latest `registry.npmjs.com`

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (keitatata) [reply yarn.lock]: sorry, I have fixed yarn.lock.

**Comment by hsluoyz** [comment]
@keitatata fix CI error:

image

and sign CLA:

image

**Comment by keitatata** [comment]
@hsluoyz
The test error is this.
```
/home/runner/work/mongoose-adapter/mongoose-adapter/node_modules/mongodb/lib/operations/add_user.js:16
        this.options = options ?? {};
                                ^

SyntaxError: Unexpected token '?'
    at wrapSafe (internal/modules/cjs/loader.js:915:[16](https://github.com/node-casbin/mongoose-adapter/actions/runs/5539023063/jobs/10111899044?pr=73#step:7:17))
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Module.replacementCompile (/home/runner/work/mongoose-adapter/mongoose-adapter/node_modules/append-transform/index.js:60:13)
    at Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
```
The MongoDB npm library uses Nullish Coalescing in its source code. 
Node.js has supported Nullish Coalescing since version 14. Therefore, if the test is executed in Node.js v12, it will definitely fail. 
Since Node.js v12 has already reached its end of life, how about running the test on v16 instead?

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**